CMakeLists.txt

```diff
@@ -90,6 +90,8 @@ else()
 	COMMAND @time -p ./tinyformat_speed_test iostreams > /dev/null
 	COMMAND @echo format timings:
 	COMMAND @time -p ./tinyformat_speed_test format > /dev/null
+	COMMAND @echo format::prepare timings:
```

Comment:
I suggest renaming `format::prepare` to `fmt::prepare` for consistency with the API naming.

Reply:
Done